bundle/manifests/lvm.topolvm.io_lvmclusters.yaml

```diff
@@ -56,6 +56,12 @@ spec:
                           description: DeviceSelector is a set of rules that should
                             match for a device to be included in the LVMCluster
                           properties:
+                            forceWipeDevicesAndDestroyAllData:
```

Comment:
I believe we should make this property immutable after creation, see https://kubernetes.io/blog/2022/09/29/enforce-immutability-using-cel/#immutability-upon-object-creation or we can use the webhook, whatever you like more